pkg/instance/service.go

```diff
@@ -184,8 +186,60 @@ func (s service) unlink(id uint) error {
 	return s.instanceRepository.Unlink(instance)
 }
 
+func (s service) matchRequiredParameters(stackParams []model.StackRequiredParameter, instanceParams []model.InstanceRequiredParameter) []string {
```

Comment:
`(s service)` is like another parameter to `matchRequiredParameters`. So we treat it like any unused parameter and remove it i.e. make it a function instead of method 😄